modules/tide_landing_page/js/hero_image_theme.js

```diff
@@ -0,0 +1,39 @@
+(function (Drupal) {
+  "use strict";
+
+  Drupal.behaviors.heroImageTheme = {
+    attach: function (context, settings) {
+      const imageTheme = document.querySelector(
```

Comment:
Use context to ensure that the behavior only attaches to elements in the current page context.